apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs

```diff
@@ -104,6 +105,43 @@ public async Task Overview()
             }
         }
 
+        [Fact]
+        public async Task CustomCallInvoker()
+        {
+            ChannelCredentials credentials = ChannelCredentials.Insecure;
+
+            // Sample: CustomCallInvoker
+            // Create a client from a custom credentials and a GcpCallInvoker that has
+            // non-default gRPC stream/channel options.
+            // Note: the GcpCallInvoker should be long-lived while the BigtableClient
+            // instances can be emphemeral and each use the same call invoker
+            BigtableServiceApiSettings settings = new BigtableServiceApiSettings
+            {
+                MaxChannels = 10,
+                PreferredMaxStreamsPerChannel = 4
+            };
+            string endpoint = BigtableServiceApiClient.DefaultEndpoint.ToString();
+            GcpCallInvoker callInvoker = new GcpCallInvoker(endpoint, credentials, settings.CreateChannelOptions());
+
+            // These will share the same set of channels to the Bigtable service.
+            BigtableClient client1 = BigtableClient.Create(callInvoker, settings);
+            // ...
+            BigtableClient client2 = BigtableClient.Create(callInvoker, settings);
+            // ...
+
+            await callInvoker.ShutdownAsync();
+
+            // Alternatively, to just create a GcpCallInvoker will standard settings, but using non-default
+            // credentials, use null or BigtableServiceApiSettings.GetDefault() for the settings
+            settings = null;
+            callInvoker = new GcpCallInvoker(endpoint, credentials, settings.CreateChannelOptions());
```

Comment:
But importantly, you can't pass in null to "new GcpCallInvoker" here, right? (This definitely *looks* like an odd call to make, until you remember it's an extension method.)

Reply:
Correct. Yes, it does look odd, but I wanted to show how it was ok. I'll add a note to specifically call it out so it's clear.